ultra/ultra/scenarios/generate_scenarios.py

```diff
@@ -444,10 +449,11 @@ def generate_social_vehicles(
         begin_time = begin_times[start_lane_id][begin_time_idx[start_lane_id]]
         begin_time_idx[start_lane_id] += 1
         end_time = begin_time + 3600  # 1 hour
+
         if "stopwatcher" in behavior_idx:
             start_lane_id = route_lanes[stopwatcher_info["direction"][0]] - 1
             end_lane_id = route_lanes[stopwatcher_info["direction"][1]] - 1
-
+            begin_time = random.randint(10, 50)
```

Comment:
Just curious, why is `begin_time` random for stopwatchers?

Reply:
Okay, so like you said, sometimes the stopwatcher wouldn't spawn before the episode ended, so this random begin time ensures that it spawns and has enough time to finish the level. Maybe we could add a comment here to explain this?

Reply:
You are correct, it might be a bit misleading without any comments. I will add an comment